config/routes.rb

```diff
@@ -43,7 +41,7 @@
   resources :holds do
     post :lift, on: :member
   end
-  resources :financial_transaction, except: %i(destroy update)
```

Comment:
Based again on the permanent_records gem I believe we can allow the destroy resource also, as permanent_records (as I noted below) overrides the destroy behavior